docs/structure-of-a-contract.rst

```diff
@@ -87,6 +90,47 @@ Function modifiers can be used to amend the semantics of functions in a declarat
         }
     }
 
+
```

Comment:
Perhaps move this section to `inheritance.rst` and link it here? This way, we would keep inheritance specs in one place.